sim/midas/src/main/cc/bridges/termination.cc

```diff
@@ -0,0 +1,64 @@
+#ifdef TERMINATIONBRIDGEMODULE_struct_guard
+
```

Comment:
The indentation in this file looks a little weird.  Expand tabs and use 2 spaces consistently.